apps/src/pythonlab/pyodideRunner.ts

```diff
@@ -56,19 +65,25 @@ export function stopPythonCode() {
 export async function runAllTests(
   source: MultiFileSource,
   dispatch: Dispatch<AnyAction>,
-  progressManager: ProgressManager | null
+  progressManager: ProgressManager | null,
+  validationFile?: ProjectFile
 ) {
-  // If the project has a validation file, we just run those tests.
-  const validationFile = Object.values(source.files).find(
-    f => f.type === ProjectFileType.VALIDATION
-  );
-  if (validationFile) {
-    // We only support one validation file. If somehow there is more than one, just run the first one.
+  // We default to using the validation file passed in. If it does not exist,
+  // we check the source for the validation file (this is the case in start mode).
```

Comment:
ah. this answers my question above about multiple validation files...

Reply:
yep there's only ever one